### ISSUE TITLE
Doesn't report all files with errors

### ISSUE THREAD

**Issue by davidgilbertson** [sasstools/sass-lint#64]
Let's say have, say, 3 files with errors. I get this in the console:
```
c:\www\mysite\node_modules\sass-lint\index.js:103
      throw new Error(result.errorCount + ' errors detected in ' + result.file
            ^
Error: 1 errors detected in app/components/ComponentTemplate/componentTemplate.scss
    at Function.sassLint.failOnError (c:\www\mysite\node_modules\sass-lint\index.js:103:13)
    at detectPattern (c:\www\mysite\node_modules\sass-lint\bin\sass-lint.js:26:10)
    at Object.<anonymous> (c:\www\mysite\node_modules\sass-lint\bin\sass-lint.js:54:3)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```
That is, because it throws an error, execution stops there.

If I change line 103 in `index.js` from this:
```js
throw new Error(result.errorCount + ' errors detected in ' + result.filePath);
```
to this
```js
console.log(result.errorCount + ' errors detected in ' + result.filePath);
```
Then I see all errors in the console:
```
1 errors detected in app/components/ComponentTemplate/componentTemplate.scss
1 errors detected in app/components/CoverTable/coverTable.scss
31 errors detected in app/components/Nav/Nav.scss
```

node v0.10.28
Windows 10
sass-lint 1.1.0

**Comment by Snugug** [comment]
We can't just log errors, we need to throw them or else the command line won't fail. If all you want to do is see what errors/warnings you have, run `sass-lint -qv`, which will prevent the command line from exiting on error and outputs verbose (formatted) view of what's going on. See the [command line interface](https://github.com/sasstools/sass-lint/tree/master/docs/cli) for all the options.

Alternatively, use one of the task runner integrations.

**Comment by davidgilbertson** [comment]
Ah, it's `-q` and `-v` combo that I was missing. Thanks.

BTW this is great and crazy fast!
